packages/docusaurus-theme-search-algolia/package.json

```diff
@@ -33,21 +33,21 @@
     "copy:watch": "node ../../admin/scripts/copyUntypedFiles.js --watch"
   },
   "dependencies": {
-    "@docsearch/react": "^3.3.3",
+    "@docsearch/react": "~3.3.3",
```

Comment:
Was this downgrade intentional @Josh-Cena ? Went from 3.5.1 to 3.3.5 but don't see any reasoning in the PR

Reply:
Looks like it was intentional according to conversations above yes.

> > @shortcuts (Sorry for the ping on a Sunday!) This looks like another Algolia type bug:
> > Error: ../node_modules/@algolia/autocomplete-plugin-algolia-insights/dist/esm/types/InsightsClient.d.ts(1,75): error TS2307: Cannot find module 'search-insights' or its corresponding type declarations.
> 
> Hey! Yes indeed there's a new option to add event tracking. I thought we shipped the peer dependency but it looks like not, (cc @shaneafsar) in the meantime you can maybe manually add it or remove the DocSearch bump so that it doesn't block you

But apparently we can now upgrade safely (https://github.com/algolia/docsearch/pull/2007) so I'll revert it